package.json

```diff
@@ -63,7 +66,9 @@
 	"jest": {
 		"collectCoverageFrom": [
 			"src/**/*.ts",
-			"!src/index.ts"
+			"!src/tmp/**/*.ts",
+			"!src/index.ts",
+			"!src/lib/Display.ts"
```

Comment:
I'm excluding `Display` from code coverage because covering it would require me to write a whole bunch of tests for what's actually core functionality, and the code base is small enough that including Display in coverage actually skews the final results pretty meaningfully.

Reply:
I _could_ add code coverage for the `UxDisplay` class, but basically what I'd be testing is that the underlying `Ux.whatever` method gets called as expected. Is that worth adding coverage for, or is it too implementation-level?

Reply:
For implementation detail things like this... you would definitely not add in unit tests for UxDisplay. But when we have proper end to end tests... all of these methods should definitely get executed. End to end tests are necessary for things like this - to ensure we didn't introduce a bug in the runtime implementation here... and so we can catch any issues if we somehow called the delegate's method incorrectly (or if it has a bug) then we'd be notified about it.

Reply:
Fair enough. Right now we don't have any E2E tests, but when we add some, I'll remove this exclusion.